Makefile

```diff
@@ -37,12 +37,15 @@ pypi_upload: install
 	$(call header,"Uploading new version to PyPi")
 	@. $(INSTALL_DIR)/bin/activate; python setup.py sdist upload -r pypi
 
-docs: install
-	$(call header,"Building docs")
+clean_docs: install
```

Comment:
Yeah.. the nice thing about this is that as we change our docstrings, running `make clean_docs` would update the documentation. 

In a future change, we can probably figure out a strategy by which we temporarily move custom documentation out of the `docs` folder, then remove the docs folder, re-generate the documentation, and move the custom documentation back in to `docs`.